### PR TITLE
domain: make schema validator test more stable

### PR DESCRIPTION
**Pull request by tiancaiamao**
I suspect the test environment is affect by the machine load, then `time.Now()` may drift from the time it's actually called.
So I extend lease to 10ms

@winkyao @zimulala

### CODE REVIEW

File: domain/schema_validator_test.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ type leaseGrantItem struct {
 func (*testSuite) TestSchemaValidator(c *C) {
 	defer testleak.AfterTest(c)()
 
-	lease := 5 * time.Millisecond
+	lease := 10 * time.Millisecond
```
Comment (zimulala) [inline domain/schema_validator_test.go]: Why change the lease's value? We still sleep a lease time at line63.

Comment (zhexuany) [reply domain/schema_validator_test.go]: I am also confused by this. May you should add a description for this PR about why do you increase lease time.

Comment (zimulala) [reply domain/schema_validator_test.go]: I think the useful changes is in ine113.

Comment (tiancaiamao) [reply domain/schema_validator_test.go]: > We still sleep a lease time at line63
Yes, I found nothing wrong with the test logic

So I change the lease's value to make it tolerant to call `time.Now()` cost and channel receive operation.

Comment (zhexuany) [reply domain/schema_validator_test.go]: Make sense

Comment (tiancaiamao) [reply domain/schema_validator_test.go]: When the lease is short, and get lease cost too long, the lease mechanism can't work.

```
leaseGrantTime  = time.Now()
<- leaseItem
```

By the time receiver get the lease, it's already stale.

Does this explain make sense? @zhexuany

**Pull request merged**